tfhe/src/high_level_api/tests/noise_squashing.rs

```diff
@@ -81,66 +81,71 @@ fn test_noise_squashing() {
 #[cfg(feature = "gpu")]
 #[test]
 fn test_gpu_noise_squashing() {
-    let config = ConfigBuilder::with_custom_parameters(
-        crate::shortint::parameters::PARAM_GPU_MULTI_BIT_GROUP_4_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M128,
-    )
-    .enable_noise_squashing(NOISE_SQUASHING_PARAM_GPU_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M128)
-    .build();
-    let cks = crate::ClientKey::generate(config);
-    let sks = cks.generate_compressed_server_key();
-
-    set_server_key(sks.decompress_to_gpu());
-
-    let mut rng = thread_rng();
-
-    // Non native type for clear
-    let clear: U256 = rng.gen();
-    let enc = FheUint256::encrypt(clear, &cks);
-    let bitand = &enc & &enc;
-
-    let squashed = bitand.squash_noise().unwrap();
-
-    let recovered: U256 = squashed.decrypt(&cks);
-    assert_eq!(clear, recovered);
-
-    // Native unsigned
-    let clear: u32 = rng.gen();
-    let enc = FheUint32::encrypt(clear, &cks);
-    let bitand = &enc & &enc;
+    let noise_squashing_params = [
+        NOISE_SQUASHING_PARAM_GPU_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M128,
+        NOISE_SQUASHING_PARAM_GPU_MULTI_BIT_GROUP_4_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M128,
```

Comment:
We should also add the multi-bit GPU noise squash parameter in the bench list of params (tfhe-benchmark/src/params_aliases.rs) and bench them please. We'll also need parameters for 128-bit compression after a multi-bit noise squash, Sam is working on it that'll be in another PR (I'll handle it).

Reply:
Indeed. I added it to the benchmark workflow. Not sure if it was in the proper way, though.